langchain/src/chat_models/openai.ts

```diff
@@ -63,7 +63,7 @@ function openAIResponseToChatMessage(
   }
 }
 
-interface ModelParams {
+export interface OpenAIInput {
```

Comment:
Because when they're exported the generated docs are a lot better. And in general it doesn't make sense to write these interfaces and then not export them. This is not a breaking change (because they weren't exported before)

Reply:
And removing the extra interface inheritance as it served no purpose, wasn't clear what args should belong to one and not the other